cost/superseded_instance_remediation/superseded_instance_remediation.pt

```diff
@@ -0,0 +1,219 @@
+# Resources
+###############################################################################
+
+resources "clouds", type: "rs_cm.clouds" do
+  filter do
+    cloud_type ne: ["soft_layer", "cloud_stack","azure","rackspace_next_gen","vscale","blue_skies","open_stack_v2","uca","open_stack_v3"]
+  end
+end
+
+###############################################################################
+# Datasources
+###############################################################################
+
+datasource "ds_instances_16" do
+  iterate(@clouds)
+  request do
+    run_script $js_instances_16, rs_cm_host, href(iter_item), rs_project_id, $param_exclude_tags
+  end
+  result do
+    encoding "json"
+    collect jmes_path(response, "[*]") do
+      field "href", jmes_path(col_item, "href")
+      field "name", jmes_path(col_item, "name")
+      field "description", jmes_path(col_item, "description")
+      field "legacy_id", to_n(jmes_path(col_item, "legacy_id"))
+      field "resource_uid", jmes_path(col_item, "resource_uid")
+      field "instance_type", jmes_path(col_item, "links.instance_type.name")
+      field "region", jmes_path(col_item, "links.cloud.region")
+      field "cloud_name", val(iter_item, "display_name")
+      field "locked", jmes_path(col_item, "locked")
+      field "state", jmes_path(col_item, "state")
+    end
+  end
+end
+
+datasource "ds_combined_instance_data" do
+  run_script $js_merge_instance_data, $ds_instances_16, $param_old_instance_type
+end
+
+###############################################################################
+# Scripts
+###############################################################################
+
+script "js_instances_16", type: "javascript" do
+  result "request"
+  parameters "rs_cm_host","cloud_href","rs_project_id","param_exclude_tags"
+  code <<-EOS
+    var filtered_tags = ""
+    if ( param_exclude_tags.length > 0 ) {
+      var filtered_tags = '&tag!=' + param_exclude_tags.join(",");
+    }
+
+    request = {
+      "auth": "auth_rs",
+      "verb": "GET",
+      "host": rs_cm_host,
+      "path": cloud_href + "/instances",
+      "headers": {
+        "X-Api-Version": "1.6",
+        "X-Account": \""+rs_project_id+\""
+      },
+      "query_params": {
+        "view": "full",
+        "filter": 'state=provisioned,operational,running'.concat(filtered_tags)
+      }
+    }
+
+  EOS
+end
+
+script "js_merge_instance_data", type: "javascript" do
+  parameters "ds_instances", "param_old_instance_type"
+  result "results"
+  code <<-EOS
+    var results = []
+    for ( i=0; i < ds_instances.length; i++) {
+      var instance = ds_instances[i]
+      if ( instance.instance_type == param_old_instance_type ){
+        results.push(instance)
+      }
+    }
+  EOS
+end
+
+###############################################################################
+# Escalations
+###############################################################################
+escalation "report_and_resize_instances" do
```

Comment:
Needs to be updated for best practices. Also, recommend removing the word "report".
Suggest `esc_resize_instances`